assembly/lib/uastc/patterns.ts

```diff
@@ -71,11 +71,11 @@ export function getThreeSubsetPattern(i: i32): i32 {
  */
 // @ts-ignore: 1206
 @inline
-export function triplicate16(v: i64): i64 {
+export function triplicate16(v: u64): u64 {
   v = (v | (v << 16)) & 0x0000FF0000FF;
-  v = (v | (v << 8)) & 0x00F00F00F00F;
-  v = (v | (v << 4)) & 0x0C30C30C30C3;
-  v = (v | (v << 2)) & 0x249249249249;
+  v = (v | (v <<  8)) & 0x00F00F00F00F;
+  v = (v | (v <<  4)) & 0x0C30C30C30C3;
+  v = (v | (v <<  2)) & 0x249249249249;
```

Comment:
Is some AS/TS-specific formatting tool used for these alignments or are they manually added?

Reply:
I usually manually added or via [vscode plugins](https://marketplace.visualstudio.com/items?itemName=wwm.better-align) if it works. If you prefer avoid such alignments I could remove this

Reply:
Personally I like alignments due to it's allow multi-line editing and also make code little bit clearer